packages/components/divider/src/DividerContent.tsx

```diff
@@ -0,0 +1,21 @@
+import { forwardRef, HTMLAttributes, PropsWithRef, ReactNode } from 'react'
+
+export interface DividerContentProps extends PropsWithRef<HTMLAttributes<HTMLSpanElement>> {
+  /**
+   * Change the component to the HTML tag or custom component of the only child.
+   */
+  asChild?: boolean
+  children?: ReactNode
+}
+
+export const DividerContent = forwardRef<HTMLSpanElement, DividerContentProps>(
+  ({ children, ...props }, ref) => {
+    return children ? (
```

Comment:
Is there a real usecase of using `Divider.Content` without any children prop?

Reply:
if someone do this

```tsx
const FOO = ({nomatherWhat}: {nomatherWhat?: string}) => {
	return (
      <>
		<h1>text</h1>
        <Divider>
          <Divider.Content>{nomatherWhat}</Divider.Content>
        <Divider>
		<p>Lorem ipsum dolor sit</p>
	 </>
}
```

User don't have to take care if `nomatherWhat` has a value or not to do not include non necessary html elements

Reply:
when there is no content provided the line is hanlded via 'before' & `after` with paddings so it that case it will have a line visually broken

Reply:
OK, but imho this kind of case should be only the responsibility of the dev, not ours... don't you think?